### PR TITLE
implement iptables resource

### PR DESCRIPTION
**Pull request by chris-rock**

### CODE REVIEW

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

File: lib/matchers/matchers.rb [hunk 1]
```diff
@@ -190,10 +190,27 @@
...
 
   failure_message do |host|
     "expected that host #{host} is resolvable"
   end
 end
+
+# matcher for iptables
+RSpec::Matchers.define :have_rule do |rule|
+  match do |tables|
+    p "have_rule matche #{rule}"
```
Comment (arlimus) [inline lib/matchers/matchers.rb]: debugging call?

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

**Comment by arlimus** [comment]
Thank you for getting iptables in! :+1: 
@delivery approve

**Comment by chef-delivery** [comment]
Change: 9dbb0f0f-be5a-43ad-9a8e-b6c21349cd1c approved by: @arlimus

**Comment by chef-delivery** [comment]
This PR has passed 'Acceptance' and is ready to be delivered!
  Use: '@delivery deliver' when validated in acceptance.

**Pull request merged**